src/main/java/com/aws/greengrass/clientdevices/auth/configuration/PolicyVariableResolver.java

```diff
@@ -44,8 +44,8 @@ public static String resolvePolicyVariables(Set<String> policyVariables, String
         }
         String substitutedFormat = format;
         for (String policyVariable : policyVariables) {
-            String attributeNamespace = policyVariableToAttributeProvider.get(policyVariable).getLeft();
-            String attributeName = policyVariableToAttributeProvider.get(policyVariable).getRight();
+            String attributeNamespace = policyVariableToAttributeProvider.get(policyVariable.toLowerCase()).getLeft();
```

Comment:
is iot core case insensitive? I'm not so sure we want to be.

Reply:
just manually tested, case-sensitive, will remove this

Reply:
done